deepchem/data/datasets.py

```diff
@@ -435,7 +436,12 @@ def __init__(self, data_dir, verbose=True):
       raise ValueError("No metadata found on disk.")
 
   @staticmethod
-  def create_dataset(shard_generator, data_dir=None, tasks=[], verbose=True):
+  def create_dataset(
+    shard_generator,
+    data_dir=None,
+    tasks=[],
+    verbose=True,
+    X_is_sparse=False):
```

Comment:
@rbharath new option for doing storing sparse matrices :)

Reply:
(party)